hack/make/gccgo

```diff
@@ -14,10 +14,12 @@ go build -compiler=gccgo \
 	-o "$DEST/$BINARY_FULLNAME" \
 	"${BUILDFLAGS[@]}" \
 	-gccgoflags "
+		-O0
 		-g
 		$EXTLDFLAGS_STATIC
 		-Wl,--no-export-dynamic
 		-ldl
+		-pthread
```

Comment:
Again, why is this necessary?  A little explanation in a comment here in the script would go a long way, if there's a good reason for this to be included.  If there's not a good reason for this to be included, then I think we shouldn't. :wink: